.drone.star

```diff
@@ -101,41 +108,42 @@ config = {
                     "webUISharingInternalGroupsToRoot",
                     "webUISharingInternalGroupsToRootEdgeCases",
                 ],
-                "oC10SharingIntGroupsSharingInd": [
-                    "webUISharingInternalGroupsSharingIndicator",
-                    "webUISharingInternalGroupsToRootSharingIndicator",
-                ],
-                "oC10SharingIntUsers": [
+                "oC10SharingIntUsers1": [
                     "webUISharingInternalUsers",
-                    "webUISharingInternalUsersCollaborator",
+                    "webUISharingInternalUsersToRootCollaborator",
+                ],
+                "oC10SharingIntUsers2": [
+                    "webUISharingInternalUsersToRoot",
+                    "webUISharingInternalUsersToRootShareWithPage",
+                ],
+                "oc10SharingIntUsers3": [
+                    "webUISharingInternalUsersToRootBlacklisted",
                     "webUISharingInternalUsersShareWithPage",
                 ],
-                "webUISharingInternalUsersBlacklisted": "oC10SharingIntUsersBlacklisted",
-                "oC10SharingIntUsersSharingInd": [
+                "oC10SharingSharingInd": [
+                    "webUISharingInternalGroupsSharingIndicator",
+                    "webUISharingInternalGroupsToRootSharingIndicator",
                     "webUISharingInternalUsersSharingIndicator",
                     "webUISharingInternalUsersToRootSharingIndicator",
                 ],
-                "oC10SharingIntUsersRoot1": [
-                    "webUISharingInternalUsersToRoot",
-                    "webUISharingInternalUsersToRootBlacklisted",
-                ],
-                "oC10SharingIntUsersRoot2": [
-                    "webUISharingInternalUsersToRootCollaborator",
+                "oC10SharingAndTrashbin": [
                     "webUISharingInternalUsersToRootPreviews",
-                    "webUISharingInternalUsersToRootShareWithPage",
+                    "webUITrashbinDelete",
+                    "webUITrashbinFilesFolders",
+                    "webUITrashbinRestore",
+                ],
+                "oC10SharingPermission": [
+                    "webUISharingPermissionsUsers",
+                    "webUISharingFolderAdvancedPermissionMultipleUsers",
                 ],
-                "webUISharingPermissionsUsers": "oC10SharingPermUsers",
                 "webUISharingPermissionToRoot": "oC10SharingPermToRoot",
-                "webUISharingPublicBasic": "oC10SharingPublicBasic",
-                "webUISharingPublicManagement": "oC10SharingPublicManagement",
-                "oC10SharingPubExpAndRoles": [
-                    "webUISharingPublicDifferentRoles",
+                "webUISharingPublic1": [
```

Comment:
Please prepend with the backend that it is running on, either "oc10..." or "oCIS..." since it's otherwise a harder task to find out which backend the pipeline failed with :)